### DISCUSSION TITLE
SESAM genie to idea statica

### DISCUSSION THREAD

**Discussion by deepaksinghal76** [idea-statica/ideastatica-public#474] [BIM API Link Disscusion] [answered]
Hi,
is it possible to extract the joints and joints forces to Idea Statica from SESAM Genie ?

**Comment by nathanlukeIDEA** [comment]
Hi @deepaksinghal76, I am not aware of a direct link available from SESAM Genie to IDEA StatiCa Checkbot or IDEA StatiCa connection. But it is something that would be nice to have. Do you have contact with SESAM Genie? we would be happy to get in touch with them to discuss a link. If SESAM can export SAF file format we can also import that directly into IDEA StatiCa.

**Reply by mpospisil** [reply, marked as answer]
IDEA StatiCa can be linked with any FE application directly - see https://github.com/idea-statica/ideastatica-public/tree/main/examples/bimapi/CheckbotBimLink
